hack/unit.sh

```diff
@@ -18,6 +18,7 @@ else
     --env DEPLOY_RAMDISK_URL=http://172.22.0.1/images/ironic-python-agent.initramfs \
     --env IRONIC_ENDPOINT=http://localhost:6385/v1/ \
     --env IRONIC_INSPECTOR_ENDPOINT=http://localhost:5050/v1/ \
+    --env OPERATOR_CONCURRENCY=3 \
```

Comment:
There might be something to be said for running 1 at a time for unit tests. Or maybe it doesn't matter at all, in which case we could just leave it out.

Reply:
Probably best to omit it.